package.json

```diff
@@ -39,6 +39,7 @@
 	"devDependencies": {
 		"@babel/core": "^7.0.0",
 		"@babel/plugin-proposal-unicode-property-regex": "^7.0.0",
+		"@babel/plugin-syntax-object-rest-spread": "^7.0.0",
```

Comment:
MDN says Chrome 60 and Firefox 55 already support this natively. We are on 69 and 62 already (maybe we need to bump the minimum supported version in manifest.json though)

Reply:
I was wondering about this. Babel errors during a build if that package or @babel/plugin-proposal-object-rest-spread are not enabled. I believe this one doesn't do the transformation, it just allows it to be parsed. I can bump the version in manifest.json but that alone doesn't seem to allow for a successful build. 

![image](https://user-images.githubusercontent.com/3723666/45248554-50093800-b2e0-11e8-90f2-bd203f4d35c4.png)